packages/shared/src/utils/platform/platform.js

```diff
@@ -55,8 +55,7 @@ export const getPlatformRedirect = routing_history => {
         return { name: platform_name.DMT5, route: routes.mt5 };
     if (isNavigationFromPlatform(routing_history, routes.smarttrader))
         return { name: platform_name.SmartTrader, route: routes.smarttrader };
-    if (isNavigationFromPlatform(routing_history, routes.cashier_p2p, true))
-        return { name: 'P2P', route: routes.cashier_p2p };
+    if (isNavegationFromP2P(routing_history, routes.cashier_p2p)) return { name: 'P2P', route: routes.cashier_p2p };
```

Comment:
`isNavigationFromP2P`?

Reply:
For P2P they want to check only for the most recent route in app_history. IsNavigationFromPlatform doesn't only check the most recent route. It checks the whole app_history which results to Back P2P button to appear all the time

Reply:
@abdalla-binary I mean the spelling of the function incorrect 😅

Reply:
ooh my bad 😅